### PR TITLE
Improve docs for get-started -> lotus

### PR DESCRIPTION
**Pull request by wanglongfei88**
As a developer who just tried the lotus for the first time, I followed the documentation to set up a lotus node,  try storing and retrieving data, and also explore different networks. Based on my experience and the confusion I had along the way, I would love to propose the following changes to improve the docs step by step.
- restructure the navigation sequence of "Get Started -> Lotus" to make the user experience more relative and clear.
- update "local installation" to 
  -  fix the formatting issue
  - add clearer instruction on build lotus for different network and check the right version of the node as well as network
  - add instruction of checking the syncing status
  - add some more instructions of lotus command
- update "switch network"
   - letting developers know different filecoin networks, and expose devnet here as well.

### CODE REVIEW

File: docs/.vuepress/config.js
Comment (mishmosh) [inline docs/.vuepress/config.js:81]: Nice re-ordering.

File: docs/get-started/lotus/installation.md
Comment (mishmosh) [inline docs/get-started/lotus/installation.md:33]: [formatting] If you split into a bullet list, the sentence before that should be a complete sentence, and end with `:` (e.g. "Users can install Lotus on Linux using any of these methods:". Or you can leave as a single sentence.

File: docs/get-started/lotus/installation.md
Comment (mishmosh) [inline docs/get-started/lotus/installation.md:83]: Thanks for the header level fixes, this is correct logically. Before merging, please check that this doesn't break sidebar formatting in the netlify preview.

File: docs/get-started/lotus/installation.md
Comment (mishmosh) [inline docs/get-started/lotus/installation.md:311]:
```suggestion
1. Checkout the release for the network you wish to use.

To join mainnet, checkout the [latest release](https://github.com/filecoin-project/lotus/releases). For networks other than mainnet...
```

Also move text from line 315 to the end of the line above. This introduces the idea of different networks first, as you suggested in our chat.

File: docs/get-started/lotus/installation.md
Comment (mishmosh) [inline docs/get-started/lotus/installation.md:338]: Regarding list number, using `1.` for each item is a way to write ordered lists in Markdown. This supports easier re-numbering in the future, if necessary. Let's preserve that since we use it across the repo.

More info: https://www.markdownguide.org/basic-syntax/#ordered-lists

File: docs/get-started/lotus/installation.md
Comment (mishmosh) [inline docs/get-started/lotus/installation.md:340]:
```suggestion
   Lotus is compiled to operate on a single network. Run one of the following commands to build lotus for your intended network.
```

A few nit-picks: 
1. Separate the two sentences with a period.
2. Networks are not lotus-specific; you can use any implementation (venus, forest, etc.) to connect to them.
3. Once the lotus application is built, you can use it to kick off multiple nodes. So technically speaking you're not building a lotus node.

File: docs/get-started/lotus/installation.md
Comment (mishmosh) [inline docs/get-started/lotus/installation.md:353]:
```suggestion
   Check that lotus is installed successfully for the right network. 
```

Shortened for clarity.

File: docs/get-started/lotus/installation.md
Comment (mishmosh) [inline docs/get-started/lotus/installation.md:400]:
```suggestion
While you wait, you can check the status of the syncing process:
```

Simplify for clarity.

File: docs/get-started/lotus/installation.md
Comment (mishmosh) [inline docs/get-started/lotus/installation.md:410]:
```suggestion
Lotus comes with built-in CLI documentation.
```

These instructions are already really long, and you have nice examples below. I made some suggestions to incorporate this content inside lines 413-423

File: docs/get-started/lotus/installation.md
Comment (mishmosh) [inline docs/get-started/lotus/installation.md:440]:
```suggestion
Or use `lotus net` to check the number of other peers that it is connected to in the Filecoin network.
```

File: docs/get-started/lotus/installation.md
Comment (mishmosh) [inline docs/get-started/lotus/installation.md:427]:
```suggestion
For example, after your Lotus daemon has been running for a few minutes, use `lotus sync` to check the sync status of your lotus node.
```

File: docs/get-started/lotus/installation.md
Comment (mishmosh) [reply docs/get-started/lotus/installation.md:400]: This sections is a duplicate of line 427. Let's keep one, not both. I prefer the placement in line 427 but up to you.

**Review by mishmosh — CHANGES_REQUESTED** [review]
Thank you for this PR! Some small detail and grammar nits, looks good generally.

In the future, please try to do 1 "thing" per PR which is much faster for review (you could split it according to your 3 bullet points above).

@johnnymatthews would you like to also take a look, or want me to follow this one to the end?

File: docs/get-started/lotus/installation.md
Comment (wanglongfei88) [reply docs/get-started/lotus/installation.md:340]: Ok, I will update.

File: docs/get-started/lotus/installation.md
Comment (wanglongfei88) [reply docs/get-started/lotus/installation.md:83]: Sidebar looks good.

File: docs/get-started/lotus/installation.md
Comment (wanglongfei88) [reply docs/get-started/lotus/installation.md]: Make sence, I will remove keep line 427 since `lotus sync wait` will also print out the status.

**Comment by mishmosh** [comment]
@wanglongfei88 Looks like a there's a build error due to a changed section name. Can you fix and see if build works again?
```
12:03:05 AM: 1 dead links was found
12:03:05 AM:   Hash is not found: [next step](#install-homebrew) (/opt/build/repo/docs/get-started/lotus/installation.md:274:59)
```

**Comment by wanglongfei88** [comment]
> @wanglongfei88 Looks like a there's a build error due to a changed section name. Can you fix and see if build works again?
> 
> ```
> 12:03:05 AM: 1 dead links was found
> 12:03:05 AM:   Hash is not found: [next step](#install-homebrew) (/opt/build/repo/docs/get-started/lotus/installation.md:274:59)
> ```

Fixed the broken link. And the build works fine now.

**Review by mishmosh — APPROVED** [review]
LGTM! In this repo we use the "author merges" convention, so go ahead and merge.

**Comment by wanglongfei88** [comment]
> LGTM! In this repo we use the "author merges" convention, so go ahead and merge.

I could not merge it. It says I am not authorized to merge this PR.

**Pull request merged**
